core/app.go

```diff
@@ -59,14 +64,17 @@ func EmptyApp() *App {
 }
 
 // LoadApp parses the commandline arguments and loads the config
-func LoadApp() (*App, error) {
+func LoadApp() (Runnable, error) {
 
 	var configFlag string
+	var renderFlag string
 	var versionFlag bool
 
 	if !flag.Parsed() {
 		flag.StringVar(&configFlag, "config", "",
 			"JSON config or file:// path to JSON config file.")
+		flag.StringVar(&renderFlag, "render", "",
+		  "- for stdout or file:// path where to save redenred JSON config file.")
```

Comment:
I recognize that it would make the parsing harder, but maybe would it more more ergonomic to have the argument to `-render` be optional? So one could do:

```
$ containerpilot -render | whatever
$ containerpilot -render -config file://containerpilot.json | whatever
$ containerpilot -render file://myfile
$ containerpilot -render file://myfile -config file://containerpilot.json
```

(I'd like to eliminate the `file://` prefix requirement here too, but that'd break backwards compatibility with the existing config flag parsing so we're stuck with it until 3.x.)

Also, we've got a typo on the help string here.